gallery/src/DarkModeModal/DarkModeModal.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useEffect, useState } from 'react';
+
+import {
+  NxButton,
+  NxModal,
+  NxFormGroup,
+  NxToggle,
+  NxFieldset,
+  NxRadio,
+  NxTooltip,
+  NxFontAwesomeIcon,
+  NxCode
+} from '@sonatype/react-shared-components';
+
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+
+import { themingEnabled, themeOverride, setThemingEnabled, setThemeOverride } from '../themeClassManager';
+
+type Props = {
+  modalCloseHandler: () => void;
+};
+
+const DarkModeModal = (props:Props) => {
+  const { modalCloseHandler } = props;
```

Comment:
Actually, I wonder if it's a good idea to rename this prop to `onClose` so it matches with the other callback names.

Reply:
> Actually, I wonder if it's a good idea to rename this prop to `onClose` or `onCancel` so it matches with the other callback names.

I updated the naming here: [0877b05](https://github.com/sonatype/sonatype-react-shared-components/pull/782/commits/0877b05ef19497dd25364eb2786a988810bf501b). I did keep the name 'onModalCloseHandler' inside pageHeader, since I think it helps distinguish what it does in the context of the page header. If you'd like me to change that one too, I can.